.github/workflows/vercel.yml

```diff
@@ -40,6 +40,8 @@ jobs:
 
       - name: Build Project Artifacts
         run: >
+          ETH_NODE_URL: ${{ secrets.ETH_NODE_URL }}
+          INFURA_ID: ${{ secrets.INFURA_ID }}
```

Comment:
FYI, this is only used in builds executed on GH Actions workflows, which are NOT the PR preview builds.